pravega/pravega-keycloak
# Issue 16: Update secrets used by Travis

**Change log description**  
Update expired Travis secrets with a new set.

**Purpose of the change**  
Fixes #16. 

**What the code does**  
Replaces secrets used to connect to Bintray. 

**How to Test**
When merged to master, the Travis CI build should automatically publish artifacts to https://oss.jfrog.org/oss-snapshot-local/io/pravega/pravega-keycloak-client/.
